api/src/repositories/observation-repository.ts

```diff
@@ -147,7 +174,6 @@ export class ObservationRepository extends BaseRepository {
         latitude::double precision,
         longitude::double precision
```

Comment:
Remove any instances of `::double precision`

Reply:
removed